pkg/query/storeset.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 type StoreSpec interface {
-	// Address for the store spec. It is used as ID for store.
+	// StoreAPI Address for the store spec. It is used as ID for store.
```

Comment:
`Addr returns the StoreAPI Address ...`?

Reply:
yes. will change comment to be go idiomatic.